samples/reducer.js

```diff
@@ -95,20 +99,40 @@ function createReducer (config) {
         }
       }, Effects.batch([
         // TODO: here is a case where ordering might be nice.
-        Effects.constant(createMeta(meta))
+        Effects.constant(createMeta(meta)),
         // Effects.constant(saveMeta(meta))
```

Comment:
as long as an undefined meta defaults to `{}`, you only need to save it when you have something to save. 

oh but yeah here you do have something to save.